Update promxy-operator readme (#64)
k0rdent/kof: promxy-operator/README.md

# promxy-operator

[Promxy](https://github.com/jacksontj/promxy) configuration is a list of serverGroup pointed to provisioned storage clusters.

> This operator dynamically builds and reloads the promxy configuration as a secret from PromxyServerGroup custom resources deployed along with storage cluster.

Reviewer: "are deployed" + a bit simpler wording, as `PromxyServerGroup` contains not just a secret:
```suggestion
This operator dynamically builds and reloads the promxy configuration as `PromxyServerGroup` custom resources are deployed along with storage cluster.
```

## Description

> This is not a generic promxy-operator, but rather an automation workaround for KOF as the promxy-config [template](promxy-operator/internal/controller/template/secret.tmpl) is limited to KOF implementation so far.

Reviewer: This file is already inside of `promxy-operator` dir, so the link is broken. Fix is:
```suggestion
This is not a generic promxy-operator, but rather an automation workaround for KOF as the promxy-config [template](internal/controller/template/secret.tmpl) is limited to KOF implementation so far.
```
